### PR TITLE
feat: support horizontal scrolling (#141)

### PR DESCRIPTION
**Pull request by zumm**
Support of horizontal scrolling. This feature uses `grid-auto-flow` css property to infer scroll mode. Demo updated.

### CODE REVIEW

**Review by rocwang — CHANGES_REQUESTED** [review]
This looks awesome! Thanks for your contribution!

Two minor things:

1. Would you mind updating the documentation in `README.md` to cover this new feature?
2. We just merged #220  , which lead to some conflicts with your PR here. Could you rebase your PR to resolve these conflicts?

File: src/demo/Control.vue [hunk 1]
```diff
@@ -68,6 +68,34 @@
       </div>
     </div>
 
+    <div :class="$style.scrollMode">
+      <p :class="$style.category">Scroll mode:</p>
+
+      <div :class="$style.radioList">
+        <label for="all" :class="$style.radioLabel">
```
Comment (rocwang) [inline src/demo/Control.vue:75]:
```suggestion
        <label for="vertical" :class="$style.radioLabel">
```

File: src/demo/Control.vue [hunk 2]
```diff
@@ -68,6 +68,34 @@
...
+            type="radio"
+            id="vertical"
+            value="vertical"
+            v-model="scrollMode"
+            :class="$style.radio"
+          />
+          Vertical
+        </label>
+
+        <label for="mens" :class="$style.radioLabel">
```
Comment (rocwang) [inline src/demo/Control.vue:86]:
```suggestion
        <label for="hirizontal" :class="$style.radioLabel">
```

**Comment by zumm** [comment]
@rocwang I can rebase, but i have no idea what i should write in `README.md`. Any suggestions?

**Review by rocwang — APPROVED** [review]

**Pull request merged**